api-test/hasura/actions/_handlers/createOrgMembers.test.ts

```diff
@@ -0,0 +1,250 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import faker from 'faker';
+
+const { mockLog } = jest.requireMock('../../../../src/common-lib/log');
+import { adminClient } from '../../../../api-lib/gql/adminClient';
+import {
+  createCircle,
+  createOrganization,
+  createProfile,
+  createOrgMember,
+  createUser,
+  mockUserClient,
+} from '../../../helpers';
+import { getUniqueAddress } from '../../../helpers/getUniqueAddress';
+
+let address, profile, user, circle, org, org_member;
+
+beforeEach(async () => {
+  address = await getUniqueAddress();
+  org = await createOrganization(adminClient);
+  circle = await createCircle(adminClient, {
+    name: 'test org',
+    organization_id: org.id,
+  });
+  profile = await createProfile(adminClient, {
+    address,
+    name: `${faker.name.firstName()} ${faker.datatype.number(10000)}`,
+  });
+  user = await createUser(adminClient, {
+    address,
+    circle_id: circle.id,
+  });
+  org_member = await createOrgMember(adminClient, {
+    org_id: org.id,
+    profile_id: profile.id,
+  });
+});
+
+describe('provided invalid input', () => {
+  test('errors if user list contains duplicates', async () => {
+    const client = mockUserClient({ profileId: profile.id, address });
```

Comment:
using mockUserClient like this is pretty neat. i would ordinarily just work with the handler and mock `req` and `res`, but this is a bit nicer because you end up having an example of a client-side call in the test, and it also verifies that the mutation is wired up to the handler in Hasura